src/client/Microsoft.Identity.Client/Internal/Requests/ISilentAuthStrategy.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.Internal.Requests
+{
+    internal interface ISilentAuthStrategy
```

Comment:
nit: maybe `ISilentAuthRequest` or `ISilentAuthRequestStrategy`?

Reply:
I like the word strategy in it because it shows that the pattern with the same name is used https://www.dofactory.com/net/strategy-design-pattern

Reply:
right, but what about `ISilentAuthRequestStrategy`?